bucket/openscad.json

```
@@ -16,8 +16,14 @@
     "extract_dir": "openscad-2021.01",
     "post_install": [
         "# OpenSCAD can't be started from a symlinked directory. See: https://github.com/openscad/openscad/issues/1309",
-        "startmenu_shortcut \"$original_dir/openscad.exe\" 'OpenSCAD'",
-        "shim \"$original_dir\\openscad.exe\" $false 'openscad'"
+        "startmenu_shortcut \"$original_dir\\openscad.exe\" 'OpenSCAD' $null $null $global",
+        "shim \"$original_dir\\openscad.exe\" $global 'openscad'"
+    ],
+    "post_uninstall": [
+        "# Remove shortcut and shim",
+        "rm_shim 'openscad' \"$(shimdir $global)\"",
+        "Remove-Item \"$(shortcut_folder $global)\\openscad.lnk\" -Force -ErrorAction SilentlyContinue",
+        "Write-Host \"Removing shortcut $(friendly_path \"$(shortcut_folder $global)\\openscad.lnk\")\""
```

```suggestion
		"Write-Host \"Removing shortcut $(friendly_path \"$(shortcut_folder $global)\\openscad.lnk\")\"",
        "Remove-Item \"$(shortcut_folder $global)\\OpenSCAD.lnk\" -Force -ErrorAction SilentlyContinue"
```
1. Better put removing before actual remove.
2. Use `OpenSCAD.lnk` for Remove-Item to avoid that the lnk could not be removed by the system for the case-sensitive file systems

---

> Use OpenSCAD.lnk for Remove-Item to avoid that the lnk could not be removed by the system for the case-sensitive file systems in case some users may enable it.

My apologies, I totally ignored this. Thanks so much for pointing that out!